spec/lib/learn_test/strategies/mocha_spec.rb

```diff
@@ -16,44 +16,47 @@
         }
       }
     end
+
     let(:runner) { double("Runner", options: {}) }
     let(:strategy) { LearnTest::Strategies::Mocha.new(runner) }
 
-    it "returns true if no node_modules directory" do
-      expect(File).to receive(:exists?).with("node_modules") { false }
-
-      expect(strategy.missing_dependencies?(package)).to eq(true)
+    context "node_modules/ does not exist" do
+      it "returns true if no node_modules directory" do
```

Comment:
Nitpick: this `context` + `it` description feel pretty duplicative

Reply:
thank you good catch. will update.